### ISSUE TITLE
Sublime Dev Build 3111+ Not Working

### ISSUE THREAD

**Issue by droath** [spectacles/CodeComplice#50]
Is anybody else having problems with CodeComplice indexing projects after updating Sublime Text to a dev build greater then version 3111?

**Comment by deluxor** [comment]
Yes, I noticed too, my sublime text is sluggish, when I start to write "if() {", when opening the brackets it breaks a lot and i can see on console that codeintel was running, I disabled it and tested again, just perfect... Please guys keep the good work but fix it 👍
